src/resources/dicts/fr.edn

```diff
@@ -803,4 +823,18 @@
     :settings-page/auto-chmod "Automatiquement changer les permissions du fichier"
     :settings-page/auto-chmod-desc "Désactiver pour permettre l'édition par plusieurs utilisateurs avec les permissions données par l'appartenance au groupe."
     :settings-page/tab-keymap "Raccourcis"
-    :unlinked-references/reference-count  (fn [total] (str total (if (= total 1) " référence non liée" " références non liées")))}
+    :unlinked-references/reference-count  (fn [total]
+                                              (cond
```

Comment:
For fn translations, we now want them to remain simple and [to only use the following fns: str, when, if and =](https://github.com/logseq/logseq/blob/master/docs/contributing-to-translations.md#editing-tips). These new rules are in place because this specific translation caused a bug that [required a separate release](https://github.com/logseq/logseq/releases/tag/0.10.5). While `cond` could be rewritten to use `if`, these are too many if branches to be considered simple and this should be reverted

Reply:
Hi,
Thanks for reviewing the PR.
I know the rule.
The bug existed due to use of undocumented `nil`. This is the limit of using uncommon format for translation.
The use of `cond` is needed if a good translation is waiting. The current translation does not give correct sentence.

Reply:
If nil is causing complex translations then feel free to file an issue or contribute a fix for the component itself. We aren't looking to maintain needlessly complex translations

Reply:
Hi,

Thanks for your comment.
I agree that complex translation is not the right way.

I wonder how can I simply translate those cases  for `:linked-references/reference-count`.

| value for `filtered-count` | value for `total` |Wanted sentence for fr.cdn | | In [en.cdn](https://github.com/logseq/logseq/blob/master/src/resources/dicts/en.edn#L229) |
|-|-|-|-|-|
| nil | 0 | Aucune référence liée | *No linked references* | "0 Linked References"
| nil | 1 | 1 référence liée | *1 linked reference* | "1 Linked Reference"
| nil | > 1 | {total} références liées | *{total} linked references* | "{total} Linked References"
| 1 | 1 | 1 référence liée / 1 | *1 linked reference / 1* | "1 of 1 Linked Reference"
| >1 | > 1 | {filtered-count} références liées / {total} | *{filtered-count} linked references / {total}* | "{filtered-count} of {total} Linked References"

I'm not comfortable with this programming language, maybe @xyhp915, @DavidLapous, @viviicat, @SylvainDuran, @jomanhattan, @sprocketc who authored on this file can help.

---
PS: you must take care that some languages use several plural forms.
Example for the definition for 3 Polish plural forms in Gettext:
`Plural-Forms: nplurals=3; plural=n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;`

Reply:
> I wonder how can I simply translate those cases for :linked-references/reference-count

There is nothing simple about handling so many branches in a translation file. My recommendation is still to revert the changes to the fn translations. These fn translations can be revisited in a later PR when nils in the components have been resolved by others

Reply:
I remove `cond` and simplified the translation, removing the case when `total` is 0.

Do you agree with the changes?

Reply:
@logseq-cldwalker is it OK for you?